pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -346,9 +347,21 @@ async def run(self):
         if self.release_version[0] < 4:
             _LOGGER.info("Don't verify payloads for OCP3 releases")
         else:
-            _LOGGER.info("Verify the swept builds match the nightlies...")
-            for _, payload in self.candidate_nightlies.items():
-                self.verify_payload(payload, advisories["image"])
+            _LOGGER.info("Verify the swept builds match the imagestreams that were updated during build-sync...")
+            self.image_stream_x_value = self.release_version[0]
+            self.image_stream_y_value = self.release_version[1]
+            self.image_stream_version = f'{self.image_stream_x_value}.{self.image_stream_y_value}'
+
+            assembly_is_base_name = assembly_imagestream_base_name_generic(self.image_stream_version, self.assembly, assembly_type)
+            arches = group_config.get("arches", [])
+            imagestreams_per_arch = [
+                payload_imagestream_namespace_and_name(default_imagestream_namespace_base_name(),
+                                                       assembly_is_base_name, arch, private=False)
+                for arch in arches
+            ]
+            for is_namespace_and_name in imagestreams_per_arch:
+                is_str = f"{is_namespace_and_name[0]}/{is_namespace_and_name[1]}"
```

Comment:
I don't think we added support for using the namespace in https://github.com/openshift-eng/art-tools/pull/1301 
So when we run oc command https://github.com/openshift-eng/art-tools/pull/1301/files#diff-39dea280c5186504ea2e9c88f6c6c85441f70a18821e9a1afbb2d9fe69f635d6R524

```
is_namespace, is_name = is.split("/")
oc ... -n {is_namespace} {is_name}
```

Reply:
ah my bad, i see it now!